### PR TITLE
[#740] Remove cheaping

### LINKED ISSUE #740: Remove cheaping in uWSGI
This is related to [Issue #1702 in uWSGI](https://github.com/unbit/uwsgi/issues/1702). Test can fail if uWSGI starts kill worker with workload.

### PR DESCRIPTION
**Pull request by kirillmakhonin**
This closes #740

### CODE REVIEW

**Comment by kirillmakhonin** [comment]
bot, test all

File: k8s/edi/uwsgi.ini [hunk 1]
```diff
@@ -3,5 +3,5 @@ module = legion.edi.wsgi
 http = 0.0.0.0:5000
 uid = www-data
 gid = www-data
-cheaper = 2
 processes = 16
+workers = 2
```
Comment (mcdoker18) [inline k8s/edi/uwsgi.ini]: Why did you add the `workers` option?
If I understand correctly, `workers` is synonim of the `processes`. `edi` server will have only 2 workers after your changes.

Comment (kirillmakhonin) [reply k8s/edi/uwsgi.ini]: Agree

**Comment by kirillmakhonin** [comment]
bot, test all
